src/elements/Button/Button.module.css

```diff
@@ -162,10 +162,10 @@
     border: solid 1px transparent;
 
     &.default {
-      color: var(--button-color);
+      color: var(--calender-date-color);
```

Comment:
is this a mistake? this would change all default Button colors and is now pointing to a calendar token